src/AutoMapper/Configuration/MemberConfigurationExpression.cs

```diff
@@ -20,7 +20,7 @@ public MemberConfigurationExpression(MemberInfo destinationMember, Type sourceTy
 
         public MemberInfo DestinationMember => _destinationMember;
 
-        public void DoNotInline()
+        public void MapAtRuntime()
         {
             PropertyMapActions.Add(pm => pm.Inline = false);

```

Comment:
I'd say no. I think for us inline makes more sense. MapAtRuntime is a little vague :)